sky/provision/azure/instance.py

```diff
@@ -184,98 +184,170 @@ def _get_head_instance_id(instances: List) -> Optional[str]:
     return head_instance_id
 
 
-def _create_instances(
-        compute_client: 'azure_compute.ComputeManagementClient',
-        resource_client: 'azure_resource.ResourceManagementClient',
-        cluster_name_on_cloud: str, resource_group: str,
-        provider_config: Dict[str, Any], node_config: Dict[str, Any],
-        tags: Dict[str, str], count: int) -> List:
+def _create_network_interface(
+        network_client: 'azure_network.NetworkManagementClient', vm_name: str,
+        provider_config: Dict[str,
+                              Any]) -> 'azure_network_objs.NetworkInterface':
+    network = azure.azure_mgmt_models('network')
+    compute = azure.azure_mgmt_models('compute')
```

Comment:
very minor nit: making the variable a bit easier to read.
```suggestion
    network_models = azure.azure_mgmt_models('network')
    compute_models = azure.azure_mgmt_models('compute')
```

Reply:
I think it's fine to keep the original way as it's a local variable and it will be less verbose